manifests/params.pp

```diff
@@ -24,7 +24,7 @@
   }
 
   # Amazon Linux already have aws cli installed:
-  if getvar('::ec2_metadata') and $::operatingsystem != 'Amazon' {
+  if getvar('::ec2_metadata') != '' and $::operatingsystem != 'Amazon' {
```

Comment:
Sorry I messed it up, there are no changes needed.
This code is working just fine. `getvar('::ec2_metadata')` returns the content of the ec2_metadata fact as it is expected.

Anyway I would suggest changing this condition. I'm not using an ec2 VM but for some reason the fact ec2_metadata is not empty. Because of this, it returns true and then the module tries to install a bunch of packages (failing if unzip is not installed....).

Would it be a problem changing the default value of `$aws_cli_install` to false, and letting the user changing it if interested?